File: tours_and_activities/v1/get/tours_and_activities_retrieve/Node SDK/tours_activities.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Extract the information about an activity with ID '56777'
    const response = await amadeus.shopping.activity("56777").get({
      north: 41.397158,
      west: 2.160873,
      south: 41.394582,
      east: 2.177181,
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();

File: tours_and_activities/v1/get/tours_and_activities_by_square/iOS SDK/ToursActivities.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

amadeus.shopping.activities.bySquare.get(params: ["north": "41.397158",
    "west": "2.160873",
    "south": "41.394582",
    "east": "2.177181"],
    onCompletion: { result in
        switch result {
        case let .success(response):
            print(response.data)
        case let .failure(error):
            fatalError(error.localizedDescription)
        }
    })
